app/browsered/field-validation-checks.js

```diff
@@ -1,5 +1,8 @@
 'use strict'
 
+// npm dependencies
+const _ = require('lodash')
```

Comment:
This is client side JS so including lodash loads the whole thing in the browser. Can you just include the function you need like so:

```js
const isNumber = require('lodash/isnumber')
```